docs/index.md

```diff
@@ -252,9 +252,6 @@ So we're going to add a statement and type in our goal statement.
 
 > In the Editor click on the
 > icon <img width="16" height="16" src="add.svg" alt="add"> (add new statement).
-> Don't try to click on the similar
-> icon <img width="16" height="16" src="duplicate.svg" alt="duplicate"> as
```

Comment:
This icon doesn't exist anymore, so there is no chance to click on the old "duplicate" button instead of the "add" button.